README.md

```diff
@@ -2,6 +2,12 @@
 
 A tool for finding the optimal set of hyperparameters for retrieval from a vector index.
 
+## I/O
+
```

Comment:
Link to the sample folder so they can see the structure of the dataset and outputs?